README.md

```diff
@@ -16,7 +16,7 @@ View the docs [here](https://hexdocs.pm/swarm).
 
 ```elixir
 defp deps do
-  [{:swarm, "~> 0.2"}]
+  [{:swarm, "~> 0.4.4"}]
```

Comment:
I prefer to pin to the minor version instead of the patch version, as it requires less upkeep.